src/ngx_http_lua_balancer.c

```diff
@@ -325,30 +325,7 @@ ngx_http_lua_balancer_get_peer(ngx_peer_connection_t *pc, void *data)
         pc->socklen = bp->socklen;
         pc->cached = 0;
         pc->connection = NULL;
-
```

Comment:
@ops-dev-cn 

the definition of `peer_name` variable should be sanitized together.